iamy/aws.go

```diff
@@ -434,6 +435,14 @@ func (a *AwsFetcher) isSkippableManagedResource(cfnType CfnResourceType, resourc
 		}
 	}
 
+	if len(a.SkipPathPrefixes) > 0 {
```

Comment:
This if block is unnecessary, since the for loop below will just do nothing if the length is zero.